model/log/sender.go

```diff
@@ -0,0 +1,210 @@
+package log
+
+import (
+	"context"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/mongodb/grip/level"
+	"github.com/mongodb/grip/message"
+	"github.com/mongodb/grip/send"
+	"github.com/pkg/errors"
+)
+
+const defaultMaxBufferSize = 1e7 //nolint: unused
```

Comment:
It seems like this _is_ used later

Reply:
Yeah this will get used once I create the exported functions, but I just didn't want the linter to complain. It trickles down, since the unexported function isn't exported it complains that this const isn't used either.